R/ForecastRegrTask.R

```diff
@@ -0,0 +1,57 @@
+#' @export
+#' @rdname Task
+#' @importFrom zoo index coredata
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
```

Comment:
Classiftask only has the `@export` and `@rdname Task` in its document section. Do you mean the imports?